src/Adapter/Meta/MetaDataProvider.php

```diff
@@ -56,6 +56,14 @@ public function getIdByPage($pageName)
         return $idMeta;
     }
 
+    /**
+     * @return array
+     */
+    public function getAvailablePages()
+    {
+        return Meta::getPages(true);
```

Comment:
Do you think query would make sense here instead? :thinking:

Reply:
ahh dont think so - if you check inside that function you will see a lot of stuff going on there ;D